main/src/ca/uwaterloo/flix/language/errors/LexerError.scala

```diff
@@ -18,6 +18,7 @@ package ca.uwaterloo.flix.language.errors
 import ca.uwaterloo.flix.language.CompilationMessage
 import ca.uwaterloo.flix.language.ast.SourceLocation
 import ca.uwaterloo.flix.util.Formatter
+import ca.uwaterloo.flix.language.phase.Lexer.{InterpolatedStringMaxNestingLevel, BlockCommentMaxNestingLevel}
```

Comment:
Perhaps its better to break this dependency. And simply say its too deep.

We don't want programmers to necessarily know such internal limits.

Reply:
Hmm, If I am a user of Flix and I get this error my first thought would be "well how deep can it go then?".
My thought is that the allowed nesting level is necessary information to fix the error.

Should we really remove it from the message?

Reply:
If you need more than 32  nesting depth something is deeply wrong.

Reply:
Like then you should talk to the compiler developers because you are doing something very strange :)

Reply:
Okay, that's fair. I will remove it 👍